app/scripts/modules/core/src/managed/overview/artifact/Constraints.less

```diff
@@ -0,0 +1,45 @@
+.Constraints {
+  margin-top: var(--s-spacing);
+  color: var(--color-dovegray);
+
+  .pending-version-constraint {
+    display: grid;
+    grid-template-columns: 14px auto;
+    column-gap: 6px;
+    align-items: center;
+
+    .constraint-title {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      color: inherit;
+
+      & > svg {
+        margin-left: var(~'--2xs-spacing');
+        padding-top: 2px;
+        transition: transform 0.15s ease;
+      }
+    }
+
+    .constraint-content {
+      margin-top: var(--s-spacing);
+      margin-bottom: var(--m-spacing);
```

Comment:
Same comment here about utilizing styleguide classes

Reply:
btw, I'm using css because it supports hot reloading and it's much faster to develop with